src/migrations/20201124173716-add-default-createdat-updatedat.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    const value = { defaultValue: Sequelize.fn('NOW'), allowNull: false, type: Sequelize.DATE };
+
+    await queryInterface.changeColumn('Users', 'createdAt', value);
```

Comment:
We might not need this. Sequelize already handles the timestamps, unless explicitly defined in the model "timestamps: false".
The only reason I can think of this would be useful is if we needed to bypass `sequelize` for some reason. But even if that was the case we could manually add the dates. Is there anything I am missing?

Reply:
Hum whenever I don't change those columns I get `ERROR: null value in column "createdAt" violates not-null constraint` running the seeder. Maybe Sequelize doesn't set the timestamp columns when doing a `bulkInsert`?

Reply:
Weird- createdAt and updatedAt are supposed to be managed by sequelize. https://sequelize.org/master/manual/model-basics.html#timestamps  Bulk operations shouldn't have an effect on this. I don't want to hold this PR up, but I would consider this tech debt that we should address later if it gets merged as is.